.github/workflows/release_swift_bindings.yml

```diff
@@ -0,0 +1,138 @@
+name: Release Swift Bindings
+
+on:
+  workflow_dispatch:
```

Comment:
Updated the release swift bindings to just go on manual triggers for now.

*Note this can be triggered before this PR is merged to main using the `gh` cli:

```
gh workflow run workflow-name --ref branch-name
```